client/src/css/App.css

```diff
@@ -1,46 +1,95 @@
 * {
-    margin: 0;
-    box-sizing: border-box;
-    font-family: 'Trebuchet MS';
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Trebuchet MS";
 }
 
 @font-face {
-    font-family: 'Trebuchet MS';
-    src: null;
+  font-family: "Trebuchet MS";
+  src: null;
 }
 
 .App {
-    height: 100vh;
-    width: 100%;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    background-image: url(../imgs/luminosity-led-logo.png);
-    background-repeat: no-repeat;
-    background-position: top;
-    background-color: whitesmoke;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url(https://wallpaperaccess.com/full/5227230.png);
```

Comment:
This could be removed as it makes it hard to determine how to change the background color.  I did that in [LL-85](https://luminosity-led.atlassian.net/jira/software/projects/LL/boards/1?selectedIssue=LL-85)